scripts/nuget-debug-helpers.ps1

```diff
@@ -203,4 +203,30 @@ Assumes the VSIXInstaller.exe is on the path.
 Function DowngradeNuGetVsix()
 {
     & VSIXInstaller.exe /d:NuGet.72c5d240-f742-48d4-a0f1-7016671e405b
+}
+
+<#
+Installs the NuGet extension build in the local repo.
+#>
+Function InstallCustomNuGetVsix()
+{
+    & VSIXInstaller.exe $(Join-Path $NuGetClientRoot "artifacts\VS15\NuGet.Tools.vsix" )
```

Comment:
Does this come up often as a need? Guessing the scenario is you want to experiment with a VSIX, but don't want to open VS and have it insert the VSIX?

Reply:
It's for scenarios where you want to test the functionality, but don't need the debugger overhead. 

I have worked a lot on solution load a lot, and in those scenarios I can't rely on the experimental instance.